### ISSUE TITLE
Glide issues (unable to find 'i' in github.com/openshift/api/route/v1)

### ISSUE THREAD

**Issue by cdrage** [redhat-developer/odo#377]
Anyone else getting these Glide issues when running `glide update --strip-vendor`?

```sh
github.com/redhat-developer/odo  coloured-output ✗                                                                                                                                                                                                                                                                                                                   8m ⚑  ⍉
▶ glide cc
[INFO]  Glide cache has been cleared.

github.com/redhat-developer/odo  coloured-output ✗                                                                                                                                                                                                                                                                                                                    8m ⚑  
▶ glide update --strip-vendor
[INFO]  Downloading dependencies. Please wait...
[INFO]  --> Fetching github.com/blang/semver.
[INFO]  --> Fetching github.com/google/go-github.
[INFO]  --> Fetching github.com/pkg/errors.
[INFO]  --> Fetching github.com/sirupsen/logrus.
[INFO]  --> Fetching github.com/spf13/pflag.
[INFO]  --> Fetching github.com/spf13/cobra.
[INFO]  --> Fetching github.com/openshift/client-go.
[INFO]  --> Fetching github.com/openshift/source-to-image.
[INFO]  --> Fetching github.com/onsi/gomega.
[INFO]  --> Fetching github.com/onsi/ginkgo.
[INFO]  --> Fetching github.com/kubernetes-incubator/service-catalog.
[INFO]  --> Fetching github.com/satori/go.uuid.
[INFO]  --> Fetching github.com/ghodss/yaml.
[INFO]  --> Fetching github.com/fatih/color.
[INFO]  --> Fetching github.com/fsnotify/fsnotify.
[INFO]  --> Setting version for github.com/ghodss/yaml to v1.0.0.
[INFO]  --> Setting version for github.com/sirupsen/logrus to v1.0.5.
[INFO]  --> Setting version for github.com/spf13/pflag to v1.0.0.
[INFO]  --> Setting version for github.com/pkg/errors to v0.8.0.
[INFO]  --> Setting version for github.com/fsnotify/fsnotify to v1.4.7.
[INFO]  --> Setting version for github.com/google/go-github to v15.0.0.
[INFO]  --> Setting version for github.com/fatih/color to v1.6.0.
[INFO]  --> Setting version for github.com/satori/go.uuid to v1.1.0.
[INFO]  --> Setting version for github.com/spf13/cobra to v0.0.1.
[INFO]  --> Setting version for github.com/blang/semver to v3.5.1.
[INFO]  --> Setting version for github.com/onsi/ginkgo to v1.4.0.
[INFO]  --> Setting version for github.com/kubernetes-incubator/service-catalog to v0.1.9.
[INFO]  --> Setting version for github.com/onsi/gomega to v1.3.0.
[INFO]  --> Setting version for github.com/openshift/client-go to release-3.9.
[INFO]  Resolving imports
[INFO]  --> Fetching golang.org/x/crypto.
[INFO]  --> Fetching golang.org/x/sys.
[INFO]  --> Fetching github.com/inconshreveable/mousetrap.
[ERROR] Error scanning github.com/openshift/api/build/v1: cannot find package "." in:
        /home/wikus/.glide/cache/src/https-github.com-openshift-api/build/v1
[ERROR] Error scanning github.com/openshift/api/apps/v1: cannot find package "." in:
        /home/wikus/.glide/cache/src/https-github.com-openshift-api/apps/v1
[ERROR] Error scanning github.com/openshift/api/image/docker10: cannot find package "." in:
        /home/wikus/.glide/cache/src/https-github.com-openshift-api/image/docker10
[ERROR] Error scanning github.com/openshift/api/image/v1: cannot find package "." in:
        /home/wikus/.glide/cache/src/https-github.com-openshift-api/image/v1
[ERROR] Error scanning github.com/openshift/api/project/v1: cannot find package "." in:
        /home/wikus/.glide/cache/src/https-github.com-openshift-api/project/v1
[ERROR] Error scanning github.com/openshift/api/route/v1: cannot find package "." in:
        /home/wikus/.glide/cache/src/https-github.com-openshift-api/route/v1
```

**Comment by cdrage** [comment]
https://github.com/Masterminds/glide/issues/945 maybe (just investigating now what I can do..)

**Comment by cdrage** [comment]
Seems that simply adding: `- package: github.com/openshift/api`

However, now I'm getting an issue with Docker:

```
[ERROR] Error scanning github.com/docker/docker/api/types/container: cannot find package "." in:
        /home/wikus/.glide/cache/src/git-github.com-openshift-moby-moby/api/types/container
```

**Comment by kadel** [comment]
I've tried to reproduce this, but no luck.
Everything works fine for me. Even after adding  `- package: github.com/openshift/api`

**Comment by cdrage** [comment]
Seems that after deleting glide completely / deleting vendor / reverting to an older version of Glide I was able to fix this. No idea what caused it however (even searching on Google people spontaniously have this error and dissapears).

I have a hunch however it was due to a GitHub redirection issue or a part of GitHub being down (I had TLS errors earlier in the day hitting their API). Anyways, closing for now!
